.github/workflows/integration-tests-rootcheck-tier-0.yml

```diff
@@ -0,0 +1,86 @@
+name: Rootcheck integration tests tier 0
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+
+  pull_request:
+    paths:
+        - ".github/workflows/integration-tests-rootcheck-tier-0.yml"
+        - "src/Makefile"
+        - "tests/integration/conftest.py"
+        - "tests/integration/test_rootcheck/**"
+
+jobs:
+  build:
+    env:
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+      BRANCH_BASE: ${{ github.base_ref || inputs.base_branch }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
```

Comment:
@TomasTurina I think this could be useful for all the tests. What do you think? or Should I remove it?

Reply:
I think we need to identify why the test stuck and solve the problem in the test itself instead of this.